### PR TITLE
Add conversion support for troy ounces

### PR DESCRIPTION
**Pull request by eliotw**

### CODE REVIEW

**Comment by cannawen** [comment]
Hey, the Troy ounce conversion looks good, but in certain subreddits they will just refer to them as "ounces" (see enhancement requirement point 2 in #30). 

Do you want to write code to process "oz" as "Troy oz" in some subreddits? If you don't feel like it, I can break that requirement out into a separate issue and accept this PR as is :) Thanks!

**Comment by eliotw** [comment]
Ah missed that. I will take a look at it and update the request.

**Comment by eliotw** [comment]
Hi @cannawen, I took a quick look. I read your suggestion in the original issue about preprocessing the input to convert 'ounces' to 'troy ounces' when appropriate. However, I don't think that can be accomplished currently because only the input string is available there. Seems like the comment's subreddit would need to be available when preprocessing the inputs, or some how associating a list of imperial units with some subreddits. Thoughts?

**Comment by cannawen** [comment]
Yes, good point. It would need a bit of a refactor (pass in a comment obejct to the preprocess instead of the comment[body]. The comment object should contain the subreddit)

**Comment by eliotw** [comment]
Hi @cannawen, updated the PR to include the refactor to preprocess. Please review when you have a chance.

File: test/converter-test.js [hunk 1]
```diff
@@ -160,6 +160,23 @@ describe('Converter', () => {
...
+      it('should convert', () => {
+        testConvert(
+          [
+            "1.25 troy ounces",
+            "1.25 oz t",
+            "5 ozt"
+          ],
+          {
+            "1.25 troy ounces" : "38.88 g",
+            "1.25 troy ounces" : "38.88 g",
```
Comment (cannawen) [inline test/converter-test.js]: Oh, one more thing I noticed (sorry for all the back-and-forth, I should have caught this earlier) is that these two keys are the same, so the bottom one will override the top one. You can see what I mean if you change line 172 to say `"1.25 troy ounces" : "0 g",` and the test will still pass

**Review by cannawen — CHANGES_REQUESTED** [review]
Hey, the refactor looks great! One small comment about the tests.

File: test/converter-test.js (hunk 1 above)
Comment (eliotw) [reply test/converter-test.js]: Updated.

**Review by cannawen — APPROVED** [review]

**Comment by cannawen** [comment]
Woohoo! It'll be deployed in the next release, v0.11.7 :) Thank you for your help!!

**Pull request merged**